vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -28,6 +30,11 @@ def get_pods(inventory)
       process_collection(inventory["pod"], :container_groups) { |n| parse_pod(n) }
     end
 
+    def get_replication_controllers(inventory)
+      process_collection(inventory["replication_controller"], :container_replication_controllers)\
```

Comment:
> Actually, use the do/end notation.

@Fryguy I think double spaces would be better. do\end notation would break consistency with other similar functions like [these](https://github.com/ManageIQ/manageiq/blob/master/vmdb/app/models/ems_refresh/parsers/kubernetes.rb#L20):

```Ruby
def get_services(inventory)
    process_collection(inventory["service"], :container_services) { |s| parse_service(s) }
end

def get_pods(inventory)
    process_collection(inventory["pod"], :container_groups) { |n| parse_pod(n) }
end
```